RELEASE/scripts/autoscend/combat/auto_combat_default_stage1.ash

```diff
@@ -147,6 +147,10 @@ string auto_combatDefaultStage1(int round, monster enemy, string text)
 		}
 
 		if (sources >= 4) {
+			if(canUse($skill[Kneebutt], false))
```

Comment:
there's no point gating this behind sources that I can see: kneebutt and headbutt are always +1 damage and a free attack

Reply:
Agreed, I didn't do this, I don't know why it was there. Cut out this conditional.

I also cut out the use of Clobber which makes this worse by skipping the DFSS damage point (not that we're generally setting up to use DFSS).